apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -129,4 +135,7 @@ export default connect(state => ({}), dispatch => ({
   removeStudent(id) {
     dispatch(removeStudent(id));
   },
+  startSavingStudent(id) {
+    dispatch(startSavingStudent(id));
+  },
```

Comment:
I believe as shorthand here you can pass an object of action creators as the `mapDispatchToProps` argument, like this:
```js
export default connect(state => ({}), {
  startEditingStudent,
  cancelEditingStudent,
  removeStudent,
  startSavingStudent,
})(ManageStudentActionsCell);
```